src/library/dlgtrackinfomulti.cpp

```diff
@@ -170,22 +172,63 @@ void DlgTrackInfoMulti::init() {
         pBox->setEditable(true);
         // We allow editing the value but we don't want to add each edit to the item list
         pBox->setInsertPolicy(QComboBox::NoInsert);
+
+        connect(pBox,
+                &QComboBox::currentIndexChanged,
+                [pBox]() {
+                    // If we have multiple value we also added the Clear All item.
+                    // If the Clear item has been selected, remove the placeholder
+                    // in order to have a safe indicator in validEditText() whether
+                    // the box has been edited.
+                    auto data = pBox->currentData(Qt::UserRole);
+                    if (data.isValid() && data.toString() == kClearItem) {
+                        pBox->lineEdit()->setPlaceholderText(QString());
+                        pBox->setCurrentIndex(-1); // This clears the edit text
+                        // Remove the Clear item afte use. If required, it's added
+                        // as first item.
+                        pBox->removeItem(0);
+                    }
+                });
     }
+    // Note: unlike other tags, comments can be multi-line, though while QComboBox
+    // can have multi-line items its Q*Line*Edit is not suitable for editing multi-
+    // line content. In order to get the same UX for comments like we have for
+    // regular tags, the two buddies require a special setup:
+    // * txtCommentBox is not editable
+    // * if an item is selected in txtCommentBox, the text is shown in txtComment
+    // * for multiple values, we show the <various> placeholder also in txtCommentBox
+    // This also requires some special handling in saveTracks().
+    txtCommentBox->setInsertPolicy(QComboBox::NoInsert);
+    connect(txtCommentBox,
+            &QComboBox::currentIndexChanged,
+            this,
+            [this](int idx) {
```

Comment:
note the clazy warning -- idx not used (so I think you can do `int _` ? )

Reply:
debug leftover, will remove